index.html

```diff
@@ -21,25 +21,33 @@
 </head>
 <body>
   <header class="header">
-    <div class="container">
+    <div class="container bar-cont flex">
       <div class="bar flex">
-        <div class="logo">
+        <div id="logo" class="logo">
           <h2>lRebornsl</h2>
         </div><!--logo-->
-        <div class="nav flex">
-          <nav class="nav-bar">
+        <div id="nav" class="aux nav hidden flex">
+          <nav class="nav-bar flex">
+            <div class="nav-item">
               <a href="#">Portfolio</a>
+              <hr>
+            </div>
+            <div class="nav-item">
               <a href="#">About</a>
+              <hr>
+            </div>
+            <div class="nav-item">
               <a href="#">Contact</a>
+            </div>
```

Comment:
- [ ] Impressive work you did here I must say 👏 . However, there are two requirements I believe you missed while implementing the interactivity of the mobile menu.

- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

## Requirements:
![image](https://github.com/lRebornsl/portfolio/assets/50990925/31006a42-5fb2-428b-b4f1-3615990cabed)

- These two functionality gives a better user experience to allow your users to navigate properly on the website. Kindly add paths to each of the sections,  also create the interactivity option missing above 👍